docs/how/delete-metadata.md

```diff
@@ -8,6 +8,10 @@ Read on to find out how to perform these kinds of deletes.
 
 _Note: Deleting metadata should only be done with care. Always use `--dry-run` to understand what will be deleted before proceeding. Prefer soft-deletes (`--soft`) unless you really want to nuke metadata rows. Hard deletes will actually delete rows in the primary store and recovering them will require using backups of the primary metadata store. Make sure you understand the implications of issuing soft-deletes versus hard-deletes before proceeding._ 
 
+## Accessing datahub CLI
+
+To use the datahub CLI you need to have the datahub Python package installed as explained in [Metadata Ingestion](../../metadata-ingestion/README.md) or you can use the `metadata-ingestion` docker image as explained in [Docker Images](../../docker/README.md). In case you are using Kubernetes you can start a pod with the metadata-ingestion docker image, get in the shell of the pod and you will have the access to datahub CLI in your kubernetes cluster.
```

Comment:
The docker image name is `linkedin/datahub-ingestion`, nor? changed at some time I think

Reply:
I did not add that as the image name. The exact image name will be linked in the `Docker Images` page.  This is just referencing that

Reply:
yeah but the image name does not appear on that documentation page. So people will need to look into code/dockerhub to find out. 
Maybe we can either update that page as well or provide the quick path here directly

Reply:
Added image name in that page. Better to have a single place in docs.

Reply:
you use two times the phrase 
> use the `metadata-ingestion` docker image as
it would be great if you can change the name of the docker image to `datahub-ingestion` as well to avoid confusion. Thanks :)

Reply:
done